lldb/tools/lldb-dap/JSONUtils.cpp

```diff
@@ -581,19 +581,19 @@ llvm::json::Value CreateStackFrame(lldb::SBFrame &frame,
 
   EmplaceSafeString(object, "name", frame_name);
 
-  auto target = frame.GetThread().GetProcess().GetTarget();
-  auto source = CreateSource(frame.GetPCAddress(), target);
```

Comment:
Would this patch become simpler if we called `StackFrame::GetFrameCodeAddressForSymbolication` instead of `GetPCAddress` here? I think that's how LLDB solves this problem